ebml_matroska.xml

```diff
@@ -520,7 +520,7 @@
   <element name="ContentSigHashAlgo" path="0*1(\Segment\Tracks\TrackEntry\ContentEncodings\ContentEncoding\ContentEncryption\ContentSigHashAlgo)" id="0x47E6" type="uinteger" minver="1" webm="0" default="0">
     <documentation lang="en">The hash algorithm used for the signature. A value of '0' means that the contents have not been signed but only encrypted. Predefined values:<br/> 1 - SHA1-160<br/> 2 - MD5</documentation>
   </element>
-  <element name="Cues" path="0*1(\Segment\Cues)" id="0x1C53BB6B" type="master" minver="1">
+  <element name="Cues" path="0*1(\Segment\Cues)" id="0x1C53BB6B" type="master" unknownsizeallowed="1" minver="1">
```

Comment:
At first I thought it was a good idea to temporarily write data and a size when it's over. But with other changes on the unknownsize that would mean the Segment needs to be temporarily with an unknownsize. Also in that case Cues may have garbage during writing. So it's probably not a good move. 

If one wants to use unknown size during writing it's fine even on elements were it's not allowed. But proper values should be written when 'finalizing' the file.